server/src/model.ts

```diff
@@ -1742,6 +1985,7 @@ export async function browseTrendingContent({
     contentViews ON contentViews.activityId = content.id
   WHERE
     content.isDeleted = FALSE
+    AND content.ownerId <> ${libraryId}
```

Comment:
Add a parameter to determine if this WHERE should be equal to libraryId or not equal. That way we could use it for the library part, too, if we want a trending section there.